src/callstack.cr

```diff
@@ -17,6 +17,19 @@ require "callstack/lib_unwind"
   require "debug/dwarf"
 {% end %}
 
+# `caller` returns the current execution stack (a backtrace) in the form
```

Comment:
Mentioning the method name is redundant and a sentence should start with a capital letter. `in the form of` is better expressed as `as`. And please use the return type `` `Array(String)` `` instead of a textual description.

I am not sure if there should be an example of a return value, because the actual formatting and content detail can vary between debug and release builds... maybe this should be mentioned then.